RingController/RingController.Designer.cs

```diff
@@ -623,10 +623,16 @@ private void InitializeComponent()
             // 
             this.division_textbox.FormattingEnabled = true;
             this.division_textbox.Items.AddRange(new object[] {
-            "Male Light",
-            "Male Heavy",
-            "Female Light",
-            "Female Heavy"});
+// Correcting the devision
+            "Children (Under  12 Years)",
+"Cadet (12-14 years)",
+"Junior (15-17 years)",
+"Senior (18-30 years)
+"Senior (31 - 40 years)",
+"Senior (41-50 Years)",
+    "Senior (51-60 years)",
+     "Senior (61-65 years)",
+"Senior (66 and above)");
```

Comment:
Does this actually work? It looks like it's missing the closing } paren of the object.